playwright/components/index.ts

```diff
@@ -1,8 +1,12 @@
 import type { Page } from "@playwright/test";
-import { ICON } from "./locators";
+import { ICON, LINK } from "./locators";
 
 const icon = (page: Page) => {
   return page.locator(ICON);
 };
 
-export default icon;
+const link = (page: Page) => {
```

Comment:
question: Is it worth including a locator for `pill` and `help` here in this shared file as it's likely they'll be used in multiple test files?